contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -3,47 +3,64 @@ package txs
 import (
 	"context"
 	"errors"
+	"flag"
 	"fmt"
 
+	"github.com/gnolang/gno/tm2/pkg/crypto"
+
 	"github.com/gnolang/gno/gno.land/pkg/gnoland"
 	"github.com/gnolang/gno/gno.land/pkg/gnoland/ugnot"
 	"github.com/gnolang/gno/tm2/pkg/bft/types"
 	"github.com/gnolang/gno/tm2/pkg/commands"
-	"github.com/gnolang/gno/tm2/pkg/crypto"
 	"github.com/gnolang/gno/tm2/pkg/std"
 )
 
 var errInvalidPackageDir = errors.New("invalid package directory")
 
-var (
-	// Keep in sync with gno.land/cmd/start.go
-	genesisDeployAddress = crypto.MustAddressFromString("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5") // test1
-	genesisDeployFee     = std.NewFee(50000, std.MustParseCoin(ugnot.ValueString(1000000)))
-)
+// Keep in sync with gno.land/cmd/start.go
+var genesisDeployFee = std.NewFee(50000, std.MustParseCoin(ugnot.ValueString(1000000)))
+
+type addPkgCfg struct {
+	txsCfg          *txsCfg
+	deployerAddress string
+}
+
+func (c *addPkgCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.StringVar(
+		&c.deployerAddress,
+		"deployer-address",
+		"g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5", // test1
+		"The address that will be used to deploy the package",
```

Comment:
Nitpick: the flag help is always lowercase, at least in the standard we follow

Reply:
fixed on c80eb7c622532c7b6ab1dd1be0cd79b317182ab2